core/src/themes/ionic.functions.font.scss

```diff
@@ -1,30 +1,29 @@
 @use "sass:math";
 
 $baselineSize: 16px !default;
-$baselineUnit: 1rem !default;
 
 /**
  * Convert a font size to a dynamic font size.
  * Fonts that participate in Dynamic Type should use
  * dynamic font sizes.
  * @param size - The initial font size including the unit (i.e. px or pt)
- * @param unit (optional) - The unit to convert to. Use this if you want to
- * convert to a unit other than $baselineUnit.
  */
-@function dynamic-font($size, $unit: $baselineUnit) {
-  @return (math.div($size, $baselineSize)) * $unit;
+@function dynamic-font($size) {
```

Comment:
Couldn't we just add the `baselineSize` as an argument also to make the `baselineUnit` work with something other than `16px`? It seems like we should handle these exceptions so that we aren't saying _only_ use the `dynamic-font` functions if the base is `16px`, otherwise write the math division that is already handled by the `dynamic-font` functions. 

We could also separate them into two different functions based on the conversion unit if that would be easier to understand. If this is out of scope for now, we should add a comment to the function explaining it only works with a base of `16px` when converting to `rem` units.

Reply:
🤔 Yeah that could work. Let me play around with that a bit.